api/rally2-matching-system/matching-system-swagger.yml

```diff
@@ -58,34 +51,25 @@ definitions:
   Template:
     type: object
     description: |
-      Object that contains a byte array of template data.
+      Data transfer object for a template.
     properties:
       Template:
-        description: A byte array that contains template data.
-        type: array
+        description: A base64 encoded string that contains template data.
```

Comment:
I thought we specifically didn't want this base 63 encoded?

Reply:
We were correct in saying that the algo service template model is a raw []byte  - however, this document specifies how that data is transferred via HTTP and how it is encoded in JSON.

As per the [json marshaling documentation](https://golang.org/pkg/encoding/json/#Marshal):

> Array and slice values encode as JSON arrays, except that []byte encodes as a base64-encoded string, and a nil slice encodes as the null JSON value.